Library/Homebrew/formula_installer.rb

```diff
@@ -1142,14 +1142,21 @@ def pour
       )
     end
 
-    tab.tap = formula.tap
+    # fill in missing/outdated parts of the tab
+    # keep in sync with Tab#to_bottle_json
+    tab.used_options = []
+    tab.unused_options = []
+    tab.built_as_bottle = true
     tab.poured_from_bottle = true
-    tab.time = Time.now.to_i
-    tab.head = HOMEBREW_REPOSITORY.git_head
-    tab.source["path"] = formula.specified_path.to_s
     tab.installed_as_dependency = installed_as_dependency?
     tab.installed_on_request = installed_on_request?
+    tab.time = Time.now.to_i
+    tab.compiler = DevelopmentTools.default_compiler
     tab.aliases = formula.aliases
+    tab.arch = Hardware::CPU.arch
```

Comment:
I'd like to include the CPU targeted by the bottle in the tab, `effective_arch` which is usually `oldest_cpu`.

Reply:
Could do but that would be another case for a field in https://github.com/Homebrew/brew/blob/ab99855c12c02eebd185ca1402c33cccf62374f6/Library/Homebrew/development_tools.rb#L101-L106